### ISSUE TITLE
Wait for center on next unit when issuing commands to settlers

### ISSUE THREAD

**Issue by kazgaroth011** [SWY1985/CivOne#91]
Again some nitpicking....

Basically when you issue orders to a settler (to e.g. build a road), the icon R (or I for iirigation etc.) should first be visible on the settler, and only then can you center in to the next waiting unit.  Similarly if this settler finishes building a road (or irrigation ...) in this same turn when you issued the command, then this road should first be visible and only then can the next unit be shown.

Basically now CivONE switches to the next unit immediately and you cannot see the icon on the settler.

**Comment by SWY1985** [comment]
I will have a go at this soon... but not high priority at the moment...

**Comment by iStruggle** [comment]
Is this possibly linked to the turn-ordering and popups/sidebar info issues?

**Comment by SWY1985** [comment]
This issue has already been resolved.
